content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/smart-irrigation-system-v2/content.md

```diff
@@ -0,0 +1,586 @@
+
+The final wiring should be similar to the following picture:
+
+![Edge Control with all the wiring](assets/PHYSICAL_CONN1.png)
+
+## Irrigation System Overview
+
+The irrigation system works as a whole: it integrates the water flow measurement and the activation of the valves, done by the Edge Control, with the Cloud communication using the MKR WAN 1310.
+
+The Edge Control is responsible for:
+
+- Measuring the water usage with a water flow sensor.
+- Measuring the soil humidity level using watermark sensors.
+- Controlling the LCD screen of the Edge Control Enclosure kit; where different system variables will be shown, including soil humidity.
+- Deciding whether to irrigate based on soil local humidity.
+
+The MKR WAN 1310 is responsible for:
+
+- Providing Cloud connectivity using LoRaWAN®.
+- Reporting the values of the Edge Control sensors on the cloud. 
+
+The communication between both devices is done leveraging the I2C communication protocol.
+
+![Smart irrigation system with Edge Control](assets/SETUP.png)
+
+### Valves Control
+
+The valves can be controlled manually by using the onboard button, one tap opens the valve one, two taps valve two, and so on. Also, the valves can be controlled automatically by the system when the soil moisture is poor. To do this the "Smart mode" has to be enabled by tapping the button five times. The working time of the valves is monitored and reported on the cloud to enable an efficient visualization of the average daily use.
+
+![Latching valves activation pulse](assets/VALVES_ACTIVATION.png)
+
+***The valves used are latching ones, which means they are activated by a single pulse. The polarity defines if it opens or closes. This pulse must be in the range of 20-40 ms, more than that time could damage the latching outputs of your device.*** 
+
+### Water Usage
+
+The water flow sensor will measure the water used and will calculate its volume in liters. This information will be monitored through the cloud and the integrated LCD.
+
+![Water flow sensor output signal](assets/FLOW_SENSOR.png)
+
+The water flow is measured by a Hall effect sensor that generates a pulsed output by the rotation steps of a propeller inside the metal body.
+
+The data is shown on dedicated widgets in the Arduino IoT Cloud.
+
+![Water usage widgets in the Arduino Cloud](assets/WATER_FLOW.png)
+
+### Soil Moisture Measurement
+
+Instead of measuring the percentage of water by volume in a given amount of soil, we will be using watermark sensors that are capable of measuring the physical force holding water in the soil. Those measurements are correlated with how difficult it is for the plants to extract water from the soil, being really interesting data for agriculture applications.
```

Comment:
```suggestion
Instead of measuring the percentage of water by volume in a given amount of soil, we will be using watermark sensors that are capable of measuring the physical force holding the water in the soil. Those measurements are correlated with the effort plants have to make to extract water from the soil, a really interesting data for agricultural applications.
```